google/cloud/bigtable/table_admin_legacy_test.cc

```diff
@@ -1229,22 +1248,22 @@ TEST_F(TableAdminTest, AsyncWaitForConsistencySimple) {
         *status = grpc::Status::OK;
       });
 
-  auto make_invoke = [](std::unique_ptr<MockAsyncCheckConsistencyResponse>& r) {
-    return [&r](grpc::ClientContext* context,
-                btadmin::CheckConsistencyRequest const& request,
-                grpc::CompletionQueue*) {
-      EXPECT_STATUS_OK(IsContextMDValid(
-          *context,
-          "google.bigtable.admin.v2.BigtableTableAdmin.CheckConsistency",
-          google::cloud::internal::ApiClientHeader()));
-      EXPECT_EQ(
-          "projects/the-project/instances/test-instance/tables/test-table",
-          request.name());
-      // This is safe, see comments in MockAsyncResponseReader.
-      return std::unique_ptr<grpc::ClientAsyncResponseReaderInterface<
-          btadmin::CheckConsistencyResponse>>(r.get());
-    };
-  };
+  auto make_invoke =
+      [this](std::unique_ptr<MockAsyncCheckConsistencyResponse>& r) {
+        return [this, &r](grpc::ClientContext* context,
+                          btadmin::CheckConsistencyRequest const& request,
+                          grpc::CompletionQueue*) {
+          EXPECT_STATUS_OK(IsContextMDValid(
+              *context,
+              "google.bigtable.admin.v2.BigtableTableAdmin.CheckConsistency"));
+          EXPECT_EQ(
+              "projects/the-project/instances/test-instance/tables/test-table",
+              request.name());
+          // This is safe, see comments in MockAsyncResponseReader.
+          return std::unique_ptr<grpc::ClientAsyncResponseReaderInterface<
+              btadmin::CheckConsistencyResponse>>(r.get());
+        };
+      };
 
```

Comment:
GitHub diff UI won't let me comment on the line, but there is another occurrence in this file we should delete:

https://github.com/googleapis/google-cloud-cpp/blob/a93026f235e99ce00daeed0e710c589c0a110fe0/google/cloud/bigtable/table_admin_legacy_test.cc#L1343

Reply:
Fixed .